submissions/medinatapanpa/landingPage/dufuna.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
The files now look a lot cleaner 👍
Please properly format this file too, so it looks good like the others